packages/Frontend/src/components/LargeCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Card, Icon, Image } from 'semantic-ui-react';
+import SealedPortion from './assets/SealedPortion.jpg';
+
+
+export default class LargeCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      starFilled: false,
+    };
+  }
+  toggleFav = () => this.setState({ starFilled: !this.state.starFilled })
+  render() {
+    const { starFilled } = this.state;
+    return (
+      <div style={{ position: 'relative', display: 'inline' }}>
+        <Card>
+          <Icon
+            name="star"
+            color={starFilled ? 'yellow' : 'whitesmoke'}
+            onClick={this.toggleFav}
+            size="huge"
+            style={{
+              color: 'whitesmoke',
+              position: 'absolute',
+              zIndex: 1,
+              right: 0,
+              top: 9,
+            }}
+          />
+          <Image src={SealedPortion} />
```

Comment:
I'm also curious how this will work with different sized images. You may want to try out a few different url's and make sure that things get resized properly. Image takes a size prop that may help you here.